server/data/data-hbase/src/test/java/org/apache/james/rrt/hbase/HBaseRecipientRewriteTableTest.java

```diff
@@ -47,6 +48,9 @@ public void setMeUp() throws Exception {
     @Override
     @After
     public void tearDown() throws Exception {
+        cluster.clearTable(new String(HDomainList.TABLE_NAME));
```

Comment:
why we need that now ?

Reply:
No idea, but it corrected the error encountered, and made sens....